src/app/modules/main/chat_section/module.nim

```diff
@@ -1433,7 +1433,6 @@ method addOrUpdateChat(self: Module,
   if belongsToCommunity and sectionId != chat.communityId or
     not belongsToCommunity and sectionId != singletonInstance.userProfile.getPubKey():
     return
-  self.updateBadgeNotifications(chat, chat.unviewedMessagesCount > 0, chat.unviewedMentionsCount, skipParentBadge = isSectionBuild)
```

Comment:
This was a pretty slow operation and doesn't seem necessary. We already call `updateParentBadgeNotifications` at the end of the chat creations and we set the `notificationCount` in the Item directly. So we were basically creating the item, and then updating the item a second time for no reason.